add more description to uninstall command during upgrade (#7930)
istio/istio.io: content/en/docs/setup/upgrade/index.md

### Uninstall old control plane

After upgrading both the control plane and data plane, you can uninstall the old control plane. 

> To uninstalls a control plane of revision `istio-1-6-5`, run:

Reviewer: ```suggestion
To uninstall a control plane of revision `istio-1-6-5`, run:
```

Author: done

{{< text bash >}}
$ istioctl x uninstall --revision istio-1-6-5
{{< /text >}}

> But if the existing control plane does not have revision label, you can uninstall it with original installation options, for example:

Reviewer: ```suggestion
If the old control plane does not have a revision label, uninstall it using its original installation options, for example:
```

Author: done
